Marker/Classes/Marker.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  Marker.swift
+//  Marker
+//
+//  Created by Htin Linn on 5/15/17.
+//  Copyright © 2017 Prolific Interactive. All rights reserved.
+//
+
+import Foundation
+
+/// Returns formatted Markdown text as an attributed string.
+///
+/// - Parameters:
+///   - markdownText: String with Markdown tags.
+///   - textStyle: Text style object containing style information.
+/// - Returns: Formatted Markdown text.
+public func attributedMarkdownString(from markdownText: String,
+                                     using textStyle: TextStyle) -> NSAttributedString {
+    guard let (parsedString, elements) = try? MarkdownParser.parse(markdownText) else {
+        return NSAttributedString(string: markdownText, textStyle: textStyle)
```

Comment:
Maybe I should just document that this is the default behavior and define a separate function that returns errors for people who want to handle the error themselves?